pymc/sampling/mcmc.py

```diff
@@ -348,6 +349,7 @@ def _sample_external_nuts(
             random_seed=random_seed,
             initvals=initvals,
             model=model,
+            var_names=var_names,
```

Comment:
Add a warning about var_names not beeing used by nutpie like we have for some other arguments above?

Reply:
Also, @aseyboldt how hard/reasonable is it to support this in nutpie?

Reply:
Perhaps it could be filtered in `nutpie.sample._trace_to_arviz`?

Reply:
Ideally we want to filter during sampling already since RAM is usually the issue, not disk-space?

Reply:
I agree, but there are no obvious hooks into the nutpie compiled model. It would require some changes on the nutpie side, by the looks of it.

Reply:
My top comment was to add a warning like these: https://github.com/pymc-devs/pymc/blob/abe7bc9c9961a2c1cc9e7d6b9fd41a851ca53c58/pymc/sampling/mcmc.py#L283-L294

Not to try to monkey-patch nutpie from the outside

Reply:
Once/if nutpie has similar functionality we can forward it from pymc?

Reply:
Let's go with the warning for now, and create an issue on nutpie for a solution.

Reply:
Shouldn't be too hard. Nutpie uses a numba function to compute all values that should appear in the trace (including the deterministics and transformed values). We should be able to just export a subset (code is around here: https://github.com/pymc-devs/nutpie/blob/main/python/nutpie/compile_pymc.py#L387)